terraform-infra-approvals/pre-shared-infrastructure.json

```diff
@@ -6,7 +6,9 @@
     {"type": "azurerm_windows_virtual_machine"},
     {"type": "azurerm_network_interface"},
     {"type": "azurerm_public_ip"},
-    {"type": "azurerm_key_vault_access_policy"}
+    {"type": "azurerm_key_vault_access_policy"},
+    {"type": "azurerm_private_endpoint"},
+    {"type": "azurerm_private_dns_zone"}
```

Comment:
private dns zones may not be deployed from teams infrastructure, you need to use central ones